.github/actions/test-server/action.yaml

```diff
@@ -175,6 +175,10 @@ runs:
         JIMM_CONTROLLER_NAME: "jimm"
         CONTROLLER_NAME: ${{ steps.lxd-controller.outputs.name }}
 
+    - name: Grant all users with add-model access to backing controller
+      run: juju jaas add-permission user-everyon@external can_addmodel controller-${{ steps.lxd-controller.outputs.name }}
```

Comment:
i know we treat `user-everyone@external` as public access.. but do we allow `user-*`, which is the openfga way of granting "public access"?